cell-properties.js

```diff
@@ -0,0 +1,193 @@
+let sheetDB = [];
```

Comment:
data of sheet which is a 2D matrix to store property object related to each cell